arviz/tests/external_tests/test_data_pyro.py

```diff
@@ -72,6 +79,38 @@ def test_inference_data_no_posterior(self, data, eight_schools_params):
         prior = Predictive(model, num_samples=500)(
             eight_schools_params["J"], torch.from_numpy(eight_schools_params["sigma"]).float()
         )
+        predictions_data = {"J": 8, "sigma": np.array([5.0, 7.0, 12.0, 4.0, 6.0, 10.0, 3.0, 9.0])}
+        predictions = Predictive(model, posterior_samples)(
+            predictions_data["J"], torch.from_numpy(predictions_data["sigma"]).float()
+        )
+        constant_data = {"J": 8, "sigma": eight_schools_params["sigma"]}
+        predictions_constant_data = {"J": 8, "sigma": predictions_data["sigma"]}
+        # only prior
+        inference_data = from_pyro(prior=prior)
+        test_dict = {"prior": ["mu", "tau", "eta"]}
+        fails = check_multiple_attrs(test_dict, inference_data)
+        assert not fails
```

Comment:
I think using

    assert not fails, "only prior: {}".format(fails)

will yield more informative error messages, not sure about formatting working though. I'll try to check this